packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2623,6 +2623,11 @@ class KeyedSubtree extends StatelessWidget {
   /// The widget below this widget in the tree.
   final Widget child;
 
+  factory KeyedSubtree.wrap(Widget child, int childIndex) {
```

Comment:
/// Return a KeyedSubtree for child with a key that's based on the child's key or childIndex.